OpenDreamRuntime/Procs/DMProc.cs

```diff
@@ -585,6 +586,9 @@ sealed class DMProcState : ProcState {
 
                     throw new Exception($"Cannot get index {index} of {indexing}");
                 }
+                case DMReference.Type.OpenDreamProcpath: {
+                    return new DreamValue(Proc.OwningType == DreamPath.Root ? "/proc/" + Proc.ToString() : Proc.ToString());
```

Comment:
This value should be a `DreamProc` instead of a string.

Reply:
for the reasoning here, see the discord